### DISCUSSION TITLE
Can we get a Discord?

### DISCUSSION THREAD

**Discussion by verdverm** [openlit/openlit#268] [General]
Many open source projects are now choosing Discord instead of Slack. I don't really have slack open but do keep discord open, so not having a presence there causes me to pause on trying this project.

**Comment by amanagarwal042** [comment]
Hey @verdverm, We do have a discord https://discord.gg/jJjew532

We have heard mixed review up until now, hence we continued with Slack but Ill make this discord as equally active as our slack community. Feel free to join and ask questions or start discussions 🚀 .

Really appreciate the feedback and would love for you try OpenLIT and out!!

**Reply by verdverm** [reply]
I asked because I didn't find the link anywhere on the website

the invitation link might be expired, I'm unable to get in

**Reply by amanagarwal042** [reply]
https://discord.gg/rjvTm6zd

This should work

**Reply by amanagarwal042** [reply]
I have added it to our README again, Will get that updated in Website too
